stable-diffusion/yamls/finetune.yaml

```diff
@@ -0,0 +1,58 @@
+run_name: stable-diffusion-finetune # Name of the  training run used for checkpointing and other logging
+seed: 42               # Random seed
+max_duration: '2000ba'    # Duration to train specified as a Time string
+grad_accum: 'auto'       # Amount of gradient accumulation, 'auto' (GPU only) means Composer will set the optimal value, set to 1 for cpu
+use_ema: false
+eval_interval: '2000ba'
+
+dataset:
+  name: "lambdalabs/pokemon-blip-captions" # other datasets can be found at https://huggingface.co/datasets?task_categories=task_categories:text-to-image
+  resolution: 512        # training image size
+  image_column: 'image'
+  caption_column: 'text'
+  mean:
+    - 0.5                 # default chanel normalization for pokemon dataset, may need to be changed for other datasets
+  std:
+    - 0.5                 # default chanel normalization for pokemon dataset, may need to be changed for other datasets
+  train_batch_size: 32    # Training total batch size (4 per 40gb A100)
+  eval_batch_size: 32
+  prompts:
+    - "Doge"
+    - "A cute bunny rabbit"
+    - "Yoda"
+    - "An epic landscape photo of a mountain"
+    - "Girl with a pearl earring"
+    - "Cute Obama creature"
+    - "Donald Trump"
+    - "Boris Johnson"
```

Comment:
I'm a little concerned by political examples (although, I know Lambda Labs used them)

Reply:
good point. I don't love them either. Had them in to validate our stuff was working. no need to keep them around.